src/App.jsx

```diff
@@ -1,44 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, setActiveTabId] = useState(tabs[0].id);
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
+  const onTabSelected = id => {
+    const validTab = tabs.find(tab => tab.id === id);
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
+    if (validTab) {
+      setActiveTabId(id);
+    } else {
+      setActiveTabId(tabs[0].id);
+    }
+  };
 
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">
-              Tab 3
-            </a>
-          </li>
-        </ul>
-      </div>
+  const displayTab = tabs.find(tab => tab.id === activeTabId) ?? tabs[0];
 
-      <div className="block" data-cy="TabContent">
-        Some text 1
-      </div>
+  return (
+    <div className="section">
+      <h1 className="title">
+        Selected tab is {displayTab.title}
+      </h1>
+
+      <Tabs
+        tabs={tabs}
+        onTabSelected={onTabSelected}
+        activeTabId={activeTabId}
+        displayTab={displayTab}
```

Comment:
Remove this prop, the test ends up ignoring it and causing the fail. We will change how `Tabs` component uses it, just for the testing